src/pages/createPost.tsx

```diff
@@ -54,6 +55,14 @@ const createPost = () => {
 
   const [startDate, setStartDate] = React.useState<Dayjs | null>(dayjs());
   const [endDate, setEndDate] = React.useState<Dayjs | null>();
+  const [selectedTags, setSelectedTags] = useState([]);
+
+  const handleAddTag = (event, tag) => {
+    if (tag.length <= 5) {
```

Comment:
1. After 5 tags it still show "Click for add tag", it looks misleading to user.
2. Does box in tags look good?
3. How user know limit the number of tags

Reply:
Since for now tag cannot be add more than 5  (but it can still selecting even though it would be discard) and the message about selecting at most 5 tags are already appears, I think when number of tag is 5 you should disable selecting tag.